src/components/Button.js

```diff
@@ -10,9 +11,15 @@ const whatStyle = (color, wide) => ({
 });
 
 class Button extends React.Component {
+  // eslint-disable-next-line no-unused-vars
+  handleClick(buttonName) {
+    // eslint-disable-next-line react/prop-types
+    return this.props.clickHandler;
+  }
+
   render() {
     return (
-      <button type="button" style={whatStyle(this.props.color, this.props.wide)} id={`btn-${this.props.buttonName}`}>
+      <button onClick={this.handleClick} type="button" style={whatStyle(this.props.color, this.props.wide)} id={`btn-${this.props.buttonName}`}>
         {this.props.buttonName}
       </button>
```

Comment:
remember what i said up, you have an incoming function as prop, and an incoming `buttonName` as prop, which you have to attach to `onClick` of each button here